lib/manager/components/reporter/containers/Timetables.js

```diff
@@ -0,0 +1,58 @@
+// @flow
+
+import { connect } from 'react-redux'
+
+import TimetableLayout from '../components/TimetableLayout'
+import { fetchRoutes } from '../../../../gtfs/actions/routes'
+import {
+  timetableDateTimeFilterChange,
+  timetablePatternFilterChange,
+  timetableRouteFilterChange,
+  timetableShowArrivalToggle,
+  timetableTimepointToggle
+} from '../../../../gtfs/actions/timetables'
+import {getTimetableData} from '../../../selectors'
+
+const mapStateToProps = (state, ownProps) => {
+  const {
+    gtfs: {
+      filter,
+      patterns,
+      routes,
+      services,
+      timetables
+    }
+  } = state
```

Comment:
Typically when I destructure I use the syntax `const {filter} = state.gtfs`.  I find that a little easier to read.

Reply:
ok, when destructuring I'll only go down one level at time